sdks/python/apache_beam/coders/standard_coders.yaml

```diff
@@ -0,0 +1,39 @@
+coder:
+  urn: "beam:coders:bytes:0.1"
```

Comment:
I was planning on getting the basic infrastructure in in this PR and then expand coverage in a future PR.